crates/bevy_winit/src/winit_windows.rs

```diff
@@ -449,3 +462,39 @@ pub fn select_monitor(
         Entity(entity) => monitors.find_entity(*entity),
     }
 }
+
+struct DisplayInfo {
+    window_physical_resolution: (u32, u32),
+    window_logical_resolution: (f32, f32),
+    monitor_name: Option<String>,
+    scale_factor: f64,
+    refresh_rate_millihertz: Option<u32>,
+}
+
+impl core::fmt::Display for DisplayInfo {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "Display information:")?;
+        write!(
+            f,
+            "  Window physicalresolution: {}x{}",
+            self.window_physical_resolution.0, self.window_physical_resolution.1
+        )?;
+        write!(
+            f,
+            "  Window logical resolution: {}x{}",
+            self.window_logical_resolution.0, self.window_logical_resolution.1
+        )?;
+        write!(
+            f,
+            "  Monitor_name: {}",
+            self.monitor_name.as_deref().unwrap_or("")
+        )?;
+        write!(f, "  Scale factor: {}", self.scale_factor)?;
+        write!(
+            f,
+            "  Refresh rate (mHz): {}",
+            self.refresh_rate_millihertz.unwrap_or(0)
+        )?;
```

Comment:
I know that the code stores the rate in millihertz, but that's such a weird unit and it would be nicer to display it in hertz with a fixed decimal point. Shouldn't be too hard to do:

```suggestion
        let millihertz = self.refresh_rate_millihertz.unwrap_or(0);
        let hertz = millihertz / 1000;
        let extra_millihertz = millihertz % 1000;
        write!(
            f,
            "  Refresh rate (Hz): {}.{:03}",
            hertz, extra_millihertz
        )?;
```